developers/weaviate/concepts/cluster.md

```diff
@@ -74,10 +74,28 @@ The groundwork to be able to re-shard has been laid by using Weaviate's Virtual
 
 ## Node Discovery
 
-Nodes in a cluster use a gossip-like protocol through [Hashicorp's Memberlist](https://github.com/hashicorp/memberlist) to communicate node state and failure scenarios.
+By default, Weaviate nodes in a cluster use a gossip-like protocol through [Hashicorp's Memberlist](https://github.com/hashicorp/memberlist) to communicate node state and failure scenarios.
 
 Weaviate - especially when running as a cluster - is optimized to run on Kubernetes. The [Weaviate Helm chart](/developers/weaviate/installation/kubernetes.md#weaviate-helm-chart) makes use of a `StatefulSet` and a headless `Service` that automatically configures node discovery. All you have to do is specify the desired node count.
 
+### FQDN for node discovery
+
+There can be a situation where IP-address based node discovery is not optimal. In such cases, you can set `RAFT_ENABLE_FQDN_RESOLVER` and `RAFT_FQDN_RESOLVER_TLD` [environment variables](../config-refs/env-vars.md#multi-node-instances) to enable fully qualified domain name (FQDN) based node discovery.
```

Comment:
I would emphasize here that this only affects the raft layer that weaviate builds on top of memberlist (aka metadata changes).
For data read/write we still base ourselves on memberlist for node ip resolution

Reply:
Updated!